cross-chain/arbitrum/deploy_l2/01_deploy_arbitrum_tbtc_token.ts

```diff
@@ -0,0 +1,39 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { ethers, getNamedAccounts, helpers } = hre
+  const { deployer } = await getNamedAccounts()
+
+  const [arbitrumTBTC, proxyDeployment] = await helpers.upgrades.deployProxy(
+    "ArbitrumTBTC",
+    {
+      initializerArgs: ["ArbitrumTBTC", "ArbTBTC"],
+      factoryOpts: { signer: await ethers.getSigner(deployer) },
+      proxyOpts: {
+        kind: "transparent",
+      },
+    }
+  )
+
+  if (hre.network.tags.etherscan) {
+    // We use `verify` instead of `verify:verify` as the `verify` task is defined
+    // in "@openzeppelin/hardhat-upgrades" to perform Etherscan verification
+    // of Proxy and Implementation contracts.
+    await hre.run("verify", {
+      address: proxyDeployment.address,
+      constructorArgsParams: proxyDeployment.args,
+    })
+  }
+
+  if (hre.network.tags.tenderly) {
+    await hre.tenderly.verify({
+      name: "ArbitrumTBTC",
+      address: arbitrumTBTC.address,
+    })
+  }
+}
```

Comment:
Does Tenderly work with Arbitrum?

Reply:
So maybe we can remove that part for now and leave a TODO to revisit later? No need to keep it if that doesn't work as expected. We may avoid some confusion in future.

Reply:
yup, added [1ddab3e](https://github.com/keep-network/tbtc-v2/pull/554/commits/1ddab3e693866595f2a9df47b32ad8b696c26c0e)